wsd/ClientSession.cpp

```diff
@@ -1113,7 +1113,7 @@ bool ClientSession::handleKitToClientMessage(const char* buffer, const int lengt
     const std::shared_ptr<DocumentBroker> docBroker = _docBroker.lock();
     if (!docBroker)
     {
-        LOG_ERR("No DocBroker to handle kit-to-client message: " << firstLine);
+        LOG_FTL("No DocBroker to handle kit-to-client message: " << firstLine);
```

Comment:
This can happen when the user closes a document, but we still have some messages from Core queued up. Certainly not fatal. It's an error, because this is unexpected, and because there might be some important notifications worth logging, but nothing fatal here.